# A Brief Market Analysis - Reverse Mortgages 

By M. Kayumov 

In [ ]:
# Load package 

import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [ ]:
# Work with google drive 

from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


### CAGR (Compounded Annual Growth Rate)

We've collected the data for the two banks providing reverse mortgages from OSFI website and complied it into a csv file. 

In [ ]:
# Load data from OSFI on Reverse Mortgages 

data = pd.read_csv('/content/drive/MyDrive/reverse_mortgages/bank_stats_homeequity.csv')

In [ ]:
# Load data - it is in X 1000 CAD 

data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3 entries, 0 to 2
Data columns (total 5 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Bank           3 non-null      object
 1   Oct. 31, 2021  3 non-null      int64 
 2   Oct. 31, 2020  3 non-null      int64 
 3   Oct. 31, 2019  3 non-null      int64 
 4   Oct. 31, 2018  3 non-null      int64 
dtypes: int64(4), object(1)
memory usage: 248.0+ bytes


We will calculate the CAGR of the reverse mortgages for the 4 years.

In [ ]:
# Create a function to calculate CAGR 

def CAGR(beg, end, years): 
  CAGR = (((end/beg)**(1/years)) - 1)*100
  return (f'{round(CAGR,2)}%')

In [ ]:
# Indicate the parameters for CAGR

beg = data['Oct. 31, 2018'].loc[2]
end = data['Oct. 31, 2021'].loc[2]
years = data.shape[1]-1

In [ ]:
# Show the CAGR 

CAGR(beg, end, years)

'12.29%'

### Population and Demographics 

We collected the demographic data from the following resources:

- https://www.bankofcanada.ca, 
- https://www.statista.com/, 
- https://www150.statcan.gc.ca/, 
- https://www.statcan.gc.ca/, 
- https://www.canada.ca/, 
- https://www.osfi-bsif.gc.ca/     

Then, we compiled the data into a file to process here, or used the data directly from those resources.

In [ ]:
# Load data - population by age and sex - from statcan

population = pd.read_csv('/content/drive/MyDrive/reverse_mortgages/Age and sex distribution of the population, 2021, Canada.csv')

In [ ]:
# Population 55+ total and % 

population_55_total = population.loc[1:9]['Male'].sum()+population.loc[1:9]['Female'].sum()
print(population_55_total)
population_55_total_pct = round(population_55_total/(population['Male'].sum() + population['Female'].sum()),4)*100
print(f'{population_55_total_pct}%')

12376036
32.36%


We've established that the population older than 55 in Canada amounts to 12.4 mln people, which is equivalent to 32.4% of the population.

In [ ]:
total_mortgage_assets_canada = 108.2*1000000000 #https://www.bankofcanada.ca/2021/05/financial-system-review-2021/

We've established that the total sum of residential mortgage assets in Canada on the balance sheets of financial institutions amounts to CAD 108.2 BN. 

In [ ]:
pct_reverse_mortgages = round(end*1000/total_mortgage_assets_canada *100,2)
print(f'{pct_reverse_mortgages} %')

4.02 %


Of all residential mortgage assets on the balance sheet of financial institutions, reverse mortgages only constitute 4.02 %, which is disproportionately small considering the share of the population to whom this product is available. 

### Income 

We've found the data that covers the population distribution by age on https://www150.statcan.gc.ca/. 

In [ ]:
# Income data # from https://www150.statcan.gc.ca/

income_data = pd.read_csv('/content/drive/MyDrive/reverse_mortgages/11100239.csv')
income_metadata = pd.read_csv('/content/drive/MyDrive/reverse_mortgages/11100239_MetaData.csv')

We've analyzed the income brackets for the population that we are interested in.

In [ ]:
# Median income in group 16+
median_income_16_plus = income_data[(income_data['Age group'] == '16 years and over') & (income_data['Sex'] == 'Both sexes') & (income_data['REF_DATE'] == 2019) 
& (income_data['GEO'] == 'Canada')& (income_data['Statistics'] == 'Median income (excluding zeros)') & (income_data['Income source'] == 'Total income')]['VALUE'].values[0]
median_income_16_plus

37800.0

The median income for Canadians older than 16 y.o. is CAD 37.8 k.

In [ ]:
# Median income in group 55-64 

median_income_55_64 = income_data[(income_data['Age group'] == '55 to 64 years') & (income_data['Sex'] == 'Both sexes') & (income_data['REF_DATE'] == 2019) & (income_data['GEO'] == 'Canada') 
& (income_data['Statistics'] == 'Median income (excluding zeros)') & (income_data['Income source'] == 'Total income')]['VALUE'].values[0]
median_income_55_64

42800.0

The median income for Canadians aged 55-64 is CAD 42.8 k.

In [ ]:
# Median income in group 65+ 

median_income_65_plus = income_data[(income_data['Age group'] == '65 years and over') & (income_data['Sex'] == 'Both sexes') & (income_data['REF_DATE'] == 2019) & (income_data['GEO'] == 'Canada') 
& (income_data['Statistics'] == 'Median income (excluding zeros)') & (income_data['Income source'] == 'Total income')]['VALUE'].values[0]
median_income_65_plus

30400.0

The median income for Canadians older than 65 y.o. is CAD 30.4 k.

In [ ]:
# Income of seniors as percentage of average 

income_pct_55_64 = (round(median_income_55_64/median_income_16_plus *100,2))
print(f'{income_pct_55_64} %')
income_pct_65_plus = (round(median_income_65_plus/median_income_16_plus *100,2))
print(f'{income_pct_65_plus} %')

113.23 %
80.42 %


The people in the age bracket from 55 to 64 earn on average 113.23% of the national average (median). Then, the income drops to 80.42% of the national average (median) after the age of 65.
These income levels place older citizens close to the middle of the income distribution in Canada. 

Assuming that by the age of 55, most Canadians have paid off their mortgages, it appears that there is an enormous business opportunity. The share of reverse mortgages in Canada is just around 4.02% of the total mortgage amount in the structure of Canadian banks' assets. It seems to be disproportionately low compared to the share of the population that the target market for this type of product constitutes: as we know, 55+ people compose 32.36% of the population (https://www.statista.com/). 

The presence of the business opportunity is supported by the rates of growth of the reverse mortgages in the past years, especially in the last 4 years, when the CAGR on average amounted to 12.29% p.a.

To grasp the maximum share of this business opportunity and to do it in an economical way, the bank should be able to guarantee the smoothness of the reverse mortgage process. An important part of the process is the adjudication step, which, as we know, takes on average 2 days. 

This indicates that, as the business grows, the bank my face two challenges:
- the need to hire more risk management stuff to approve the mortgage requests;
- difficulty in setting the pace in the industry without the necessary automation technology in place.